.buildkite/pipeline.yml

```diff
@@ -99,7 +99,33 @@ steps:
       CMAKE_FLAGS: "-DCMAKE_TOOLCHAIN_FILE=cmake/darwin-x86_64.cmake"
       RUN_TESTS: "false"
 
-  - label: "Java Integration Tests"
+  - label: "Build & test :cpp: for Windows_x86_64-RelWithDebInfo :windows:"
+    timeout_in_minutes: 180
+    agents:
+      provider: gcp
+      machineType: "c2-standard-16"
+      minCpuPlatform: "Intel Cascade Lake"
+      image: family/ml-cpp-1-windows-2016
+    commands:
+      - 'echo "checkout succeeded"'
+      - "& Get-ComputerInfo"
```

Comment:
If `Get-ComputerInfo` is a built-in PowerShell command then should it be called without the leading `&`? I don't know if that's needlessly running it in a different invocation of PowerShell for example.

Reply:
Removed it as it was transient debug anyway